### PR TITLE
Update file requests for Honeytrap

### PR DESCRIPTION
**Pull request by nordinvannes**

### CODE REVIEW

File: scripter/scripter.go [hunk 1]
```diff
@@ -175,3 +172,76 @@ func ReloadScripts(s Scripter) {
 		}
 	}
 }
+
```
Comment (Icety) [inline scripter/scripter.go:200]: Can you place comments or split this into subfunctions?

File: utils/files/get.go [hunk 2]
```diff
@@ -69,5 +70,39 @@ func Download(url string, path string) error {
 		return err
 	}
 
+	return nil
+}
+
+// Put puts content on the place of the path
```
Comment (Icety) [inline utils/files/get.go:76]: Is there some check on whether these files are allowed to be deleted? What if a call comes in that says to delete honeytrap.go?

**Review by Icety — CHANGES_REQUESTED** [review]

File: scripter/scripter.go (hunk 1 above)
Comment (nordinvannes) [reply scripter/scripter.go:200]: Yes!

File: utils/files/get.go (hunk 2 above)
Comment (nordinvannes) [reply utils/files/get.go:76]: Done!

File: scripter/scripter.go (hunk 1 above)
Comment (nordinvannes) [reply scripter/scripter.go:200]: Done!

File: utils/files/get.go (hunk 2 above)
Comment (nordinvannes) [reply utils/files/get.go:76]: Done!

**Comment by Icety** [comment]
LGTM

**Review by Icety — APPROVED** [review]

**Pull request merged**